Dockerfile-server

```diff
@@ -5,10 +5,8 @@ FROM ghcr.io/astral-sh/uv:python3.12-bookworm AS python-builder
 
 WORKDIR /app
 
-# No wheel for indirect pycairo dependency so need build env for it to install
 RUN apt-get update && \
-    apt-get -y upgrade && \
-    apt-get -y install --no-install-recommends libcairo2-dev
+    apt-get -y upgrade
```

Comment:
Do we even need this on a regular day, when not installing any extras? Shouldn't the base image be good, for the most part?

Reply:
I don't think so, I'll just remove them. I feel it's even better without, more reproducible 👍🏻